content/acra/configuring-maintaining/general-configuration/acra-keys/_index.md

```diff
@@ -6,18 +6,19 @@ weight: 6
 
 ## acra-keys
 
-`acra-keys` is a command-line utility used for different keys operations
-especially for `v2` keystore. Each functionality is implemented as a separate subcommand:
+`acra-keys` is a command-line utility used for different keys operations for `v1`/`v2` keystores. 
+
+Each functionality is implemented as a separate subcommand:
 
 * [`list`](/acra/configuring-maintaining/general-configuration/acra-keys/list/) - lists available keys in the keystore;
 
 * [`generate`](/acra/configuring-maintaining/general-configuration/acra-keys/generate/) - generates new keys;
 
 * [`read`](/acra/configuring-maintaining/general-configuration/acra-keys/read/) - reads and prints key material in a plaintext;
 
-* [`export`](/acra/configuring-maintaining/general-configuration/acra-keys/export/) - exports keys from the keystore;
+* [`export`](/acra/configuring-maintaining/general-configuration/acra-keys/export/) - exports keys from the keystore (supports only `v2` keystore for versions before 0.95.0);
```

Comment:
```suggestion
* [`export`](/acra/configuring-maintaining/general-configuration/acra-keys/export/) - exports keys from the keystore (`v1` keystore is supported since 0.95.0 ,`v2` keystore is supported since 0.90.0);
```

i suggest to make support versions clear